vpr/src/base/netlist_writer.cpp

```diff
@@ -1818,6 +1818,14 @@ class NetlistWriterVisitor : public NetlistVisitor {
                 if (impl_input_net_id) {
                     //If there is a valid net connected in the implementation
                     AtomNetId logical_net_id = atom_ctx.nlist.port_net(port_id, orig_index);
+                    // TODO: remove after debugging is finished
```

Comment:
Please remove the `VTR_ASSERT` and just upgrade the `VTR_LOG` to a `VTR_FATAL_ERROR`?